league_rpc_linux/__main__.py

```diff
@@ -13,13 +15,19 @@
     check_league_client_process,
     player_state,
 )
+
 from league_rpc_linux.reconnect import discord_reconnect_attempt
+from league_rpc_linux.processes import LCU_Thread
```

Comment:
Why import an entire module? would it not make more sence to import a function/class that we then call somewhere in our main loop?

If you import the entire module here, i believe it will execute lcu-driver before we even run the main loop.. which could be problematic

Reply:
changed to import only the necessary function